network-store-server/src/main/java/com/powsybl/network/store/server/NetworkStoreRepository.java

```diff
@@ -407,34 +407,65 @@ public void deleteNetwork(UUID uuid, int variantNum) {
         }
     }
 
-    @SuppressWarnings("javasecurity:S5145")
-    public void cloneNetwork(UUID uuid, int sourceVariantNum, int targetVariantNum, String targetVariantId) {
+    public void cloneNetwork(UUID uuid, UUID sourceNetworkUuid, List<String> targetVariantIds) {
```

Comment:
If one the variant id does not exist like clone(...,; ..., List.of("InitialVariant, "RealVariant", "toto"). It will silently not clone toto. We should at least add a warning.